types/resize-observer-browser/index.d.ts

```diff
@@ -9,16 +9,33 @@ interface Window {
     ResizeObserver: typeof ResizeObserver;
 }
 
+interface ResizeObserverObserveOptions {
+    /**
+     * Sets which box model the observer will observe changes to. Possible values
+     * are `content-box` (the default), and `border-box`.
+     *
+     * @default "content-box"
```

Comment:
Can you use single quotes in the `@default` section?
`'content-box'`

Reply:
sure thing. applying that now